package-lock.json

```diff
@@ -1,4629 +0,0 @@
-{
```

Comment:
If you are using yarn you shouldn't have a `package-lock.json` but you should have a `yarn.lock` file

Reply:
Right, and that's what this PR does. Am I misunderstanding?

Reply:
sorry I thought this file was still there. Ignore me